app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert split_integer(8, 1) == [8]
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(6, 2) == [3, 3]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
```

Comment:
does not this test should have `1` as the second parameter?
 ```python
split_integer(17, 1) == [17]
```

Reply:
fix 2 tests. sorry for commit name, I accidentally